### ISSUE TITLE
Asynchronous use of your library

### ISSUE THREAD

**Issue by andrei-polukhin** [ping/instagram_private_api#308]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- Rlease read them **carefully** and answer completely.
- Do not post screenshots of error messages or code.
- Put an `x` into all the boxes [ ] relevant to your issue (==> [x] *no* spaces).
- Use the *Preview* tab to see how your issue will actually look like.
- Issues about reverse engineering is out of scope and will be closed without response.
- Any mention of spam-like actions or spam-related tools/libs/etc is strictly **not allowed**.

---

### Before submitting an issue, make sure you have:
- [x] Updated to the latest version v1.6.0
- [x] Read the [README](https://github.com/ping/instagram_private_api/blob/master/README.md) and [docs](https://instagram-private-api.readthedocs.io/en/latest/)
- [x] [Searched](https://github.com/ping/instagram_private_api/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] Reviewed the sample code in [tests](https://github.com/ping/instagram_private_api/tree/master/tests) and [examples](https://github.com/ping/instagram_private_api/tree/master/examples)

### Which client are you using?
- [x] app (``instagram_private_api/``)
- [ ] web (``instagram_web_api/``)

---

Hello, I have a question: can I use your library asynchronously (for example, searching users simultaneously) and not get blocked on Instagram? Will it look suspicious?

The thing is I am building a programme simplifying finding information about a person, so sometimes I need to search for information about 5-8 people, and simultaneous API calls will save a lot of time.

**Comment by andrei-polukhin** [comment]
@ping Checked by myself, it works poorly, as I get the traceback:
`instagram_private_api.errors.ClientThrottledError: Please wait a few minutes before you try again.`

I shall close the issue and will use only synchronous API calls.
